dashboard/test/factories.rb

```diff
@@ -674,7 +674,7 @@
     end_date nil
     course nil
     subject nil
-    properties {{}}
+    properties {{fixed_payment: 50}}
```

Comment:
I should have caught this earlier, but you don't need to specify all these nil attributes. Only provide default factory values that need to be different from the default in the model. You can always pass attributes to factory girl `build` or `create` methods that don't have defaults. And since properties is handled nicely by `SerializedProperties` (see my other comment on that), these can be specified directly. Instead of hard-coding dates, I prefer relative dates for tests for readability. Your call on this one. 

This can be rewritten as:

```ruby
factory :pd_payment_term, class: 'Pd::PaymentTerm' do
  start_date {Date.today}
  fixed_payment 50
end
```